util/Logger.js

```diff
@@ -6,34 +6,33 @@ const moment = require("moment");
 
 exports.log = (content, type = "log") => {
   const timestamp = `[${moment().format("YYYY-MM-DD HH:mm:ss")}]:`;
-    switch (type) {
-      case "log": {
-        return console.log(`${timestamp} ${chalk.bgBlue(type.toUpperCase())} ${content} `);
-      }
-      case "warn": {
-        return console.log(`${timestamp} ${chalk.black.bgYellow(type.toUpperCase())} ${content} `);
-      }
-      case "error": {
-        return console.log(`${timestamp} ${chalk.bgRed(type.toUpperCase())} ${content} `);
-      }
-      case "debug": {
-        return console.log(`${timestamp} ${chalk.green(type.toUpperCase())} ${content} `);
-      }
-      case "cmd": {
-        return console.log(`${timestamp} ${chalk.black.bgWhite(type.toUpperCase())} ${content}`);
-      }
-      case "ready": {
-        return console.log(`${timestamp} ${chalk.black.bgGreen(type.toUpperCase())} ${content}`);
-      } 
-      default: throw new TypeError("Logger type must be either warn, debug, log, ready, cmd or error.");
+  switch (type) {
+    case "log": {
+      return console.log(`${timestamp} ${chalk.bgBlue(type.toUpperCase())} ${content} `);
     }
-  } 
+    case "warn": {
+      return console.log(`${timestamp} ${chalk.black.bgYellow(type.toUpperCase())} ${content} `);
+    }
+    case "error": {
+      return console.log(`${timestamp} ${chalk.bgRed(type.toUpperCase())} ${content} `);
+    }
+    case "debug": {
+      return console.log(`${timestamp} ${chalk.green(type.toUpperCase())} ${content} `);
+    }
+    case "cmd": {
+      return console.log(`${timestamp} ${chalk.black.bgWhite(type.toUpperCase())} ${content}`);
+    }
+    case "ready": {
+      return console.log(`${timestamp} ${chalk.black.bgGreen(type.toUpperCase())} ${content}`);
+    }
+    default: throw new TypeError("Logger type must be either warn, debug, log, ready, cmd or error.");
+  }
 } 
 
-exports.error = (...args) => return this.log(...args, "error");
+exports.error = (...args) => this.log(...args, "error");
```

Comment:
`this.log` accepts literally two parameters, being the second given. You only need to pass the first. Same for the three next exports.